crates/rbuilder/src/live_builder/block_output/relay_submit.rs

```diff
@@ -102,14 +102,17 @@ pub struct SubmissionConfig {
     pub dry_run: bool,
     pub validation_api: ValidationAPIClient,
 
-    pub optimistic_enabled: bool,
-    pub optimistic_signer: BLSBlockSigner,
-    pub optimistic_max_bid_value: U256,
-    pub optimistic_prevalidate_optimistic_blocks: bool,
-
+    pub optimisitic_config: Option<OptimisticConfig>,
```

Comment:
Previously `optimistic` config fields were mandatory, what's the rationale making them optional?

Reply:
The relay optimistic functionality is optional, only enabled if `optimistic_enabled` is set to true. When it is set to false, those fields have default null values. That to me is kind of the same as an Option config.